gbp/patch_series.py

```diff
@@ -42,7 +45,10 @@ class Patch(object):
     def __init__(self, path, topic=None, strip=None):
         self.path = path
         self.topic = topic
-        self.strip = strip
+        if strip:
```

Comment:
I would use
    if strip is not None:

That would allow strip level of 0 to be valid